content/applications/services/fsm/default_warehouse.rst

```diff
@@ -0,0 +1,51 @@
+===============================
+Set up a user default warehouse
+===============================
+
+Setting up a **default warehouse** can be useful for field technicians who keep a supply in their
+van or those who always resupply in the same warehouse.
+
+With this feature configured, the products in sales orders created during field interventions are
+always pulled from the same warehouse, keeping the inventory accurate.
```

Comment:
Products in sales orders created during field interventions are always pulled from the same warehouse, keeping the inventory accurate with the :guilabel:` default warehouse1 feature in Odoo.